packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/java/challengeintegrationsteps.md

```diff
@@ -0,0 +1,53 @@
+### 1 - 4: Sign in and select authenticator
+
+The challenge flow follows the same first four steps as the [enrollment flow](#integrate-sdk-for-authenticator-enrollment):
+
+* Build a sign-in page on the client
+* Authenticate the user credentials
+* Handle the response from the sign-in flow
+* Display a list of possible authenticator factors
+
+### 5: Check authenticator status
+
+When the user selects the Google Authenticator factor, call `selectAuthenticator()`, to tell Okta which factor is being used.
+
+> **Note:** For Email or SMS factors, this sends a message to the user.
```

Comment:
This note doesn't seem necessary as this isn't a general guide. We're only talking about the Google factor,, not email or SMS